test/chainable.test.js

```diff
@@ -16,10 +16,8 @@ test('chainable standalone', (t) => {
       done()
     }).ready(function () {
       t.pass('we are ready')
-    }).use(function (ctx, opts, done) {
-      t.pass('3rd plugin')
-      done()
```

Comment:
Can you restore this?

Reply:
`ready` no longer returns `this` so the test now verifies that it is not chainable.

If you are concerned about testing that `use` after `ready` does the right thing I believe other tests do this, `ready adds at the end of the queue` from `after-and-ready.test.js` for example calls ready before calling use.

Please let me know if I'm misunderstanding your finding.